lib/http.js

```diff
@@ -0,0 +1,215 @@
+/*!
+ * Copyright (c) 2019-2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const {asyncHandler} = require('bedrock-express');
+const bedrock = require('bedrock');
+//const {config} = bedrock;
+const {ensureAuthenticated} = require('bedrock-passport');
+//const {validate} = require('bedrock-validation');
+const {util: {BedrockError}} = bedrock;
+
+const vcIssuer = require('./issuer');
+
+bedrock.events.on('bedrock-express.configure.routes', app => {
+  //const cfg = config['bedrock-vc-issuer'];
+  const cfg = {
+    routes: {
+      instances: '/vc-issuer/instances',
+      instance: '/vc-issuer/instances/:instanceId',
+      instanceIssuer: '/vc-issuer/instances/:instanceId/issuer',
+      automatedIssue: '/vc-issuer/issue',
+      getUser: '/vc-issuer/instances/:instanceId/users',
+      claimUser: '/vc-issuer/instances/:instanceId/claim-user'
+    }
+  };
+  const {routes} = cfg;
+
+  // create a new issuer instance
+  app.post(
+    routes.instances,
+    ensureAuthenticated,
+    // TODO: validate instance
+    asyncHandler(async (req, res) => {
+      // verifiable presentation with issuer VC and configuration and
+      // assertionMethod zcaps is required to register an issuer
+      const {id, name, controller} = req.body;
+
+      // disallow any authenticated user to act as another controller
+      const {actor} = req.user;
+      if(actor.id !== controller) {
+        throw new BedrockError(
+          'Permission denied.',
+          'NotAllowedError', {httpStatusCode: 400, public: true});
+      }
+
+      // create instance
+      const instance = {
+        id,
+        name,
+        controller
+      };
+      const record = await vcIssuer.instances.create({instance});
+      res.json(record);
+    }));
+
+  // get one or more issuer instances
```

Comment:
this is a bit atypical of our past with HTTP APIs, usually separate APIs for get one and getMany, is this a conscious decision.

Reply:
Ah, I remember why this happened now. It used to be doing a look up by `issuer` but this changed recently during a big rewrite to just be an `id`. Previously, instances did not have IDs. So this should change now that they do and things work differently. Filed #8